python/sgl_jax/srt/layers/layernorm.py

```diff
@@ -171,3 +171,36 @@ def _normalize(
         args.append(bias)
     dtype = dtypes.canonicalize_dtype(*args, dtype=dtype)
     return jnp.asarray(y, dtype)
+
+
+class GemmaRMSNorm(nnx.Module):
```

Comment:
GemmaRMSNorm did not call the maybe_distributed_mean function. When tp hidden_dim was partitioned, did it only compute the local variance?